package.json

```diff
@@ -44,12 +44,8 @@
 		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha -- --timeout 4s",
 		"ci:test:realsvc:local": "pnpm run -r --no-sort --stream --no-bail test:realsvc:local:report",
 		"ci:test:realsvc:local:coverage": "c8 --no-clean pnpm recursive --no-sort --stream --no-bail run test:realsvc:local:report",
-		"ci:test:realsvc:local:full": "pnpm run -r --no-sort --stream --no-bail test:realsvc:local:report:full",
-		"ci:test:realsvc:local:full:coverage": "c8 --no-clean pnpm recursive --no-sort --stream --no-bail run test:realsvc:local:report:full",
 		"ci:test:realsvc:tinylicious": "pnpm run -r --no-sort --stream --no-bail test:realsvc:tinylicious:report",
 		"ci:test:realsvc:tinylicious:coverage": "c8 --no-clean pnpm run -r --no-sort --stream --no-bail test:realsvc:tinylicious:report ",
-		"ci:test:realsvc:tinylicious:full": "pnpm run -r --no-sort --stream --no-bail test:realsvc:tinylicious:report:full",
-		"ci:test:realsvc:tinylicious:full:coverage": "c8 --no-clean pnpm run -r --no-sort --stream --no-bail test:realsvc:tinylicious:report:full",
```

Comment:
These four scripts became obsolete because the build-client and test-stability pipelines now only run their corresponding non-`:full` versions. The `:full` scripts now only exist in `packages/test/test-end-to-end-tests/package.json` since that package itself is the only one that needs to run them (when it runs in the e2e tests pipeline).